modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -1,5 +1,10 @@
 definition {
 
+	@summary = "Assert button is not clickable"
+	macro CannotContinue {
```

Comment:
Hey @MGabriellaS, I think that on this macro you can use the generic path that I mentioned, and put a variable on button name, so we can use in different cases, instead of just on continue button. The macro name could be '`viewDisabledButton`' also. What do you whink?

Reply:
Yeah, I agree. In this way we can use in other cases.